toolbox/ci2gubernator/README.md

```diff
@@ -45,7 +45,10 @@ The binary `ci_to_gubernator` constructs these files and uploads these artifacts
 
 ## Usage
 
-At the start of a build, execute the following command to create and upload `started.json` on GCS.
+At the start of a build, execute the following command to create and upload `started.json` on GCS. If the job is running as part of the presubmit, one should
```

Comment:
perhaps fix the line breaks to make the paragraph consistent?

Reply:
Done